src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -140,6 +140,13 @@ FUZZ_TARGET(coinselection)
     if (result_srd) result_srd->AddInputs(utxos_set, subtract_fee_outputs);
     if (result_knapsack) result_knapsack->AddInputs(utxos_set, subtract_fee_outputs);
     if (result_bnb) result_bnb->AddInputs(utxos_set, subtract_fee_outputs);
+
+    std::vector<OutputGroup> new_group_pos;
+    GroupCoins(fuzzed_data_provider, utxos, coin_params, /*positive_only=*/true, new_group_pos);
+    auto new_srd = SelectCoinsSRD(new_group_pos, target, coin_params.m_change_fee, fast_random_context, MAX_STANDARD_TX_WEIGHT);
+    if (new_srd && result_srd) {
+        new_srd->Merge(result_srd.value());
+    }
```

Comment:
In d72c0926:

Should check the `Merge` outcome. e.g. the result target and weight need to be the sum of the two merged targets and weights. Moreover, if one result uses the effective value and the other one not, the `use_effective` field must be updated.

Reply:
Done